<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gScreen</class>
 <widget class="QMainWindow" name="gScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gScreen</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>521</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="gChartLayout"/>
   </widget>
   <widget class="QPushButton" name="pb_addChart">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Chart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_delChart">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Chart</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
